### DISCUSSION TITLE
WARNING:pyrogram.connection.connection:timed out

### DISCUSSION THREAD

**Discussion by ziyingjie** [Dineshkarthik/telegram_media_downloader#64] [Q&A] [answered]
**INFO:pyrogram.session.auth:Start creating a new auth key on DC2
INFO:pyrogram.connection.connection:Connecting...
WARNING:pyrogram.connection.connection:timed out
INFO:pyrogram.connection.connection:Connecting...
WARNING:pyrogram.connection.connection:timed out
INFO:pyrogram.connection.connection:Connecting...
WARNING:pyrogram.connection.connection:timed out
WARNING:pyrogram.connection.connection:Connection failed! Trying again...

**Comment by Dineshkarthik** [comment, marked as answer]
@ziyingjie This is related to your network, make sure your network is not having any issues and then re-try.
